server/main_server/src/main.cpp

```diff
@@ -338,8 +378,9 @@ auto main(int _argc, char* _argv[]) -> int
 
             log_stacktrace_files(stfp_time_start);
             remove_leftover_agent_info_files_for_ips();
-            launch_agent_factory(__func__, write_to_stdout, enable_test_mode);
+            launch_agent_factory(__func__, write_to_stdout, enable_test_mode, access_time_resolution.data());
             migrate_and_launch_delay_server(write_to_stdout, enable_test_mode, dsm_time_start);
+            apply_access_time_updates();
```

Comment:
could put timing information around the call site here?     or perhaps within the function itself?

Reply:
See https://github.com/irods/irods/pull/8269#issuecomment-2725796466.